<!-- photo 기능 창을 위한 ui -->
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoWindow</class>
 <widget class="QMainWindow" name="PhotoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1494</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="turnButton">
       <property name="text">
        <string>회전</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="revLRButton">
       <property name="text">
        <string>좌우반전</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="revUDButton">
       <property name="text">
        <string>상하반전</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="roiButton">
       <property name="text">
        <string>사진자르기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sharpButton">
       <property name="text">
        <string>선명하게</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="grayButton">
       <property name="text">
        <string>흑백</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="huiButton">
       <property name="text">
        <string>휘도</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blurButton">
       <property name="text">
        <string>블러링</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mosaicButton">
       <property name="text">
        <string>모자이크</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brightButton">
       <property name="text">
        <string>밝기 조절</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outlineButton">
       <property name="text">
        <string>외곽선따기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(&quot;C:/Users/rkdms/김가은_영상처리/기말/imageUpload.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(&quot;C:/Users/rkdms/김가은_영상처리/기말/save.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="originButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(&quot;C:/Users/rkdms/김가은_영상처리/기말/original.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="plusButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>705</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Plus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>735</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="brButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>710</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bright</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="darkButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>740</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(&quot;C:/Users/rkdms/김가은_영상처리/기말/home.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1494</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
